.github/workflows/basic-ci-amd64.yaml

```diff
@@ -28,6 +28,8 @@ jobs:
       CONTAINERD_VERSION: ${{ matrix.containerd_version }}
       GOPATH: ${{ github.workspace }}
       KATA_HYPERVISOR: ${{ matrix.vmm }}
+      #the latest containerd from 2.0 need to set the CGROUP_DRIVER for e2e testing
```

Comment:
I think that `export CGROUP_DRIVER=""` not enough for the containerd e2e tests, to avoid us having to make a workflow change as well. Given that the PR doesn't test the workflow changes, that also suggests it is enough?

Reply:
Hi. @stevenhorsman, yes, since the workflow wouldn't be active in this PR's CI, so I had set the CGROUP_DRIVER in the tests/integration/cri-containerd/integration-tests.sh file too.

Once this pr merged, we can rm it from tests/integration/cri-containerd/integration-tests.sh.

Reply:
I'm wondering if it would be cleaner to leave the `export CGROUP_DRIVER=""` in tests/integration/cri-containerd/integration-tests.sh and remove it from the workflows as there it is just in one single place (and close to what needs it)?

Reply:
Good idea. I'd removed the CGROUP_DRIVER setting from the workflows and only kept it in the integration-tests.sh file.

Reply:
Great. Thanks!